core/src/main/java/io/th0rgal/oraxen/OraxenPlugin.java

```diff
@@ -119,6 +121,10 @@ public void onEnable() {
         RecipesManager.load(this);
         invManager = new InvManager();
         ArmorEquipEvent.registerListener(this);
+        if (Settings.REIMPLEMENT_NOTEBLOCK_FEATURES.toBool()) {
+            Bukkit.getPluginManager().registerEvents(new TuneBlockListeners(), this);
+            CustomBlockData.registerListener(this);
```

Comment:
Ithink i removed this ages ago due to some conflicts or issues, does it serve any purpose or?

Reply:
CustomBlockData.registerListener(this) makes it so that we dont have to listen for block move events (from pistons) to move the pdc data or break events to remove the pdc

Reply:
I know that ive disabled this in the past for some reasons but idk

Reply:
tbh I don't really see it having any issues with persistent data container, should work just fine